ckanext/data_qld_theme/templates/page.html

```diff
@@ -13,4 +13,19 @@
 <!-- End Google Tag Manager -->
 
     {{ super() }}
-{% endblock %}
\ No newline at end of file
+{% endblock %}
+
+
+{% block flash %}
+    <div class="global-alert" style="background-color: #f18d0e; color: #000;">
+        <div class="header">
+            <h2 id="landmark-label7">Maintenance windows</h2>
+            <time datetime="2019-07-10T17:00:00+10:00">Wednesday 10 July, 5-9PM AEST</time>
```

Comment:
ensure we don't remove this until all updates are complete

Reply:
Yes, that's the intent.